elliottlib/bzutil.py

```diff
@@ -566,6 +567,11 @@ def get_flaw_bugs(self, bug_ids: List, strict: bool = True, verbose: bool = Fals
 
 class JIRABugTracker(BugTracker):
     JIRA_BUG_BATCH_SIZE = 50
+    FIELD_BLOCKED_BY_BZ = 'customfield_12322152'  # "Blocked by Bugzilla Bug"
+    FIELD_TARGET_VERSION = 'customfield_12323140'  # "Target Version"
```

Comment:
Could you change https://github.com/openshift-eng/elliott/blob/6ae6135170dbd78c6b5918fca14c5b2e1f80bfa3/elliottlib/cli/find_bugs_kernel_cli.py#L258 as well?